.github/workflows/test.yml

```diff
@@ -49,6 +49,8 @@ jobs:
       run: |
         pip install -r dev-requirements.txt
         pip install -r requirements.txt
+        pip install --no-warn-conflicts jinja2==2.10.1
```

Comment:
@ThrawnCA this will be solved with upcoming requirement changes in frictionless-py. We just added this to not block the tests on this branch.

@aivuk would be good to sort this out on frictionless-py. IIRC it's just loosening up the requirements